Formula/leela-zero.rb

```diff
@@ -0,0 +1,41 @@
+class LeelaZero < Formula
+  desc "Neural Network Go engine with no human-provided knowledge"
+  homepage "https://zero.sjeng.org/"
+  # pull from git tag to get submodules
+  url "https://github.com/leela-zero/leela-zero.git",
+      :tag      => "v0.16",
+      :revision => "4fd6e694ead45b7cbb14ea76c5bdfcbfa662304b"
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+
+  resource "network" do
+    url "https://zero.sjeng.org/networks/2d46c3273204ba8e41bc2daffd61c42da275401321f4f7a62e45f26472b7c8c2.gz", :using => :nounzip
+    sha256 "732229be158bfb795f558b32fd9f84d28675d21cd6559794e3eabbd28840d0d8"
+  end
+
+  def install
+    mkdir "build"
+    cd "build" do
+      system "cmake", "-DUSE_CPU_ONLY=1", ".."
```

Comment:
By the way, this is a temporary measure that I hope to overcome in the near future - I'm still trying to figure out a good way to detect whether the machine has an OpenCL-compatible GPU or not. The engine runs _much_ better on GPU-enabled Macs, so it would be great to only set this flag if absolutely necessary.

But for now this at least lets it be installed everywhere.